constants.py

```diff
@@ -3,9 +3,9 @@
 """Signals and constants used in the application."""
 
 MUTE_CHECKBOX = 'mute_checkbox'
-MUTE_ALL_STREAMS = 'mute_all_streams'
-ADD_NEW_STREAM = 'add_new_stream'
-EXPORT_STREAMS_TO_CLIPBOARD = 'export_streams_to_clipboard'
+MUTE_ALL_STREAMS = 'MuteAllStreams'
```

Comment:
Why are these camel-cased now? Imo it looks better underscore-separated. But if this is done for some Qt functional reason then maybe `MUTE_CHECKBOX` should also be changed?